core/data/src/main/java/kr/co/cotton/core/data/news/repository/DefaultNewsRepository.kt

```diff
@@ -0,0 +1,30 @@
+package kr.co.cotton.core.data.news.repository
+
+import androidx.paging.Pager
+import androidx.paging.PagingConfig
+import androidx.paging.PagingData
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.flow.*
```

Comment:
🚫 **[ktlint]** <no-wildcard-imports> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Wildcard import (cannot be auto-corrected)